hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import TeamInfoTable from "components/teamDetail/TeamInfoTable/TeamInfoTable";
+
+import { RouteComponentProps, useHistory, useLocation } from "react-router-dom";
+import Header from "components/general/Header/Header";
+
+export interface PageParams {
+    id: string;
+}
+
+const TeamDetail = ({ match }: RouteComponentProps<PageParams>) => {
+    const history = useHistory();
```

Comment:
Based on the above comments, you shouldn't need to pass in any of the route page props since the TeamInfoTable component should only contain the table and not any other code.

Reply:
@YuYing-Liang see the above comment on the test file. basically whenever i write `<TeamInfoPage />` i get this error about missing props. this was my solution from a stackoverflow post

Reply:
yeah thats because TeamInfoTable doesn't need these props, you can remove them from the component (see comments in TeamInfoTable.tsx)